<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">boarder-image : url(:/newPrefix/스테이크치즈.jpg);

background: rgb(0, 151, 67);</string>
  </property>
  <widget class="QPushButton" name="btnmenu1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>150</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/newPrefix/이탈리안비엠티.png);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/menu/이탈리안비엠티.png</normaloff>images/menu/이탈리안비엠티.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>110</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmenu5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>150</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/newPrefix/치킨데리야끼.jpg);
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/menu/치킨데리야끼.jpg</normaloff>images/menu/치킨데리야끼.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>110</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmenu2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>150</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/newPrefix/에그마요.jpg);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/menu/에그마요.jpg</normaloff>images/menu/에그마요.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>110</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmenu3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>150</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/newPrefix/스테이크치즈.jpg);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/menu/스테이크치즈.jpg</normaloff>images/menu/스테이크치즈.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>110</height>
    </size>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmenu4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>150</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/newPrefix/써브웨이클럽.jpg);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/menu/써브웨이클럽.jpg</normaloff>images/menu/써브웨이클럽.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>110</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>서브웨이 클럽
7,100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>131</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>치킨 데리야끼
7,000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>에그마요
5,500</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>스테이크 &amp; 치즈
7,900</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>611</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>이탈리안 비엠티
6,700</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>670</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>670</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255);
color: rgb(255, 200, 13);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;TT.WICH&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="bmenubar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;메뉴를 선택해 주세요&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images/menu/menu.qrc"/>
 </resources>
 <connections/>
</ui>
